### ISSUE TITLE
Genetics in New TG Sucks

### ISSUE THREAD

**Issue by cpm311** [HippieStationArchive/HippieStation13-2#28]
Although I feel that Genetics was overpowered I think with the new rebase to updated TG it has been hit with quite a nerf. Now the powers that are discovered (which sometimes takes entire rounds to find) only last for about 2 minutes after injection. So now Geneticists cant actually use the powers effectively but also dont have access to the rest of medbay (so they cant save people) and they dont have access to Science (cant research). In my personal opinion, if this time restriction for powers remains, genetics should at least get some extra access to compensate for the nerf.

**Comment by KittyKathy** [comment]
I'll take a look at this to see just how bad the nerfs are, they seem really like what were you thinking bad based on what you said. If it is that bad I'll see to it that it gets reverted lets make genetics great again. laser eyes for geneticist traitor soon.

**Comment by KittyKathy** [comment]
Okay, I see your point have you read this (https://tgstation13.org/wiki/Guide_to_genetics) by any chance?

According to that you can now do this instead of injecting yourself with the DNA injector "By using the Occupant - Delayed button, the scanner will apply the buffer's content onto the next person who enters the scanner. This is mainly useful for self-experimenting. " so you can user the scanner on yourself.

It also says this " It's important to note that unlike using the scanner, injectors have a 100% chance to activate power blocks, but have a limited duration; this is true of both SE and UI/UE injectors. If you want permanent powers, you need to activate them with the scanner. " so it can be permanent just need to use the scanner on yourself with the occupant delay button.

So in conclusion you should use the occupant delay button instead of injectors but when you know you have a power block but it will not activate use the injector to see what power it is do this and figure out all the powers then spam the scanner occupant delay button on yourself to make it permanent and be a superhero.

There should also be more powers we can add because there are only like 6 now kinda boring we have to port more. also geneticist should have access to medbay and research may be a mapping or code error on the job role or just something tg did.

**Comment by JohnGinnane** [comment]
So leave /tg/ genetics as is or use our old stuff? New powers would be nice but need ideas for them and perhaps something else to activate them, rather than just "click the box till you hit jackbot"

**Comment by cacogen** [comment]
either the powers should last as long as they used to or they shouldn't take anywhere between 10 and 50 minutes to find imo

**Comment by Spacedong** [comment]
>they shouldn't take anywhere between 10 and 50 minutes to find imo

"COME GET HULK AT GENETICS I JUST FOUND IT 5 MINUTES INTO THE ROUND"

nigger no

**Comment by cacogen** [comment]
something more reliable in the medium range then

**Comment by cpm311** [comment]
Yes well in that case here is a list of **non-op powers**
Obviously this list is meant to be a stepping stone in the right direction, and is not balanced.
Please feel free to add "powers" or add input on this current list. 

**What is not included on this list:** What value is needed for an ability to show (More OP powers need higher values)

**-Lux** = The ability to emit a faint light without needing a flashlight (Equivalent to a penlight, cannot be turned off)

**-Minor Radiation Healing** = As the title says, useful for testing in genetics. (Hint Hint)

**-Electric Resistance** = This ability acts identical to insulated gloves.

**-Increased Metabolism** = Allows Food, drink, and toxins to be processed faster in the body. (To balance this I would make it only slightly increase these times)

**-(Name Pending)** = Allows you to fake your death, obviously after you became 'reanimated' you would have to dodge the crew for the effect to actually work.

**Comment by tortellinitony** [comment]
we have mutant eyeballs and butts and stuff, maybe we could have that as a mutation

**Comment by JohnGinnane** [comment]
Those are neat ideas, but we could try to remove the length they exist
